components/eventing-controller/pkg/backend/jetstreamv2/jetstream_internal_unit_test.go

```diff
@@ -372,7 +346,7 @@ func TestVlad_CheckNATSSubscriptionsCount(t *testing.T) {
 			name:                 "unexpected empty subscriptions map should result into an error",
 			givenSubscription:    subWithType,
 			givenSubscriptionMap: map[SubscriptionSubjectIdentifier]Subscriber{},
-			wantErr:              &ErrMissingNATSSubscription{},
+			wantErr:              &backenderrors.ErrMissingNATSSubscription{},
```

Comment:
I think it's good to have an errors package, since it's easier to distinct them and structure. We followed the same pattern for skip.go error. I refactored them a bit though.